pkg/config/config.go

```diff
@@ -67,13 +67,14 @@ type Config struct {
 
 // KubernetesConfig contains all Kubernetes related config items.
 type KubernetesConfig struct {
-	Kubeconfig         string             `json:"kubeconfig" yaml:"kubeconfig"`
-	ResyncInterval     types.TimeDuration `json:"resync_interval" yaml:"resync_interval"`
-	AppNamespaces      []string           `json:"app_namespaces" yaml:"app_namespaces"`
-	ElectionID         string             `json:"election_id" yaml:"election_id"`
-	IngressClass       string             `json:"ingress_class" yaml:"ingress_class"`
-	IngressVersion     string             `json:"ingress_version" yaml:"ingress_version"`
-	ApisixRouteVersion string             `json:"apisix_route_version" yaml:"apisix_route_version"`
+	Kubeconfig          string             `json:"kubeconfig" yaml:"kubeconfig"`
+	ResyncInterval      types.TimeDuration `json:"resync_interval" yaml:"resync_interval"`
+	AppNamespaces       []string           `json:"app_namespaces" yaml:"app_namespaces"`
+	ElectionID          string             `json:"election_id" yaml:"election_id"`
+	IngressClass        string             `json:"ingress_class" yaml:"ingress_class"`
+	IngressVersion      string             `json:"ingress_version" yaml:"ingress_version"`
+	WatchEndpointSlices bool               `json:"watch_endpoint_slices" yaml:"watch_endpoint_slices"`
```

Comment:
Should add the configuration for (k8s.io/api/discovery/v1)LabelManagedBy ? or reuse the field `IngressClass`?

Reply:
We only care `EndpointSlices` managed by `endpointslice-controller.k8s.io`.

```
endpointslice.kubernetes.io/managed-by: endpointslice-controller.k8s.io
```